docs/source/models/causes/maternal_disorders/gbd_2021_mncnh/obstructed_labor.rst

```diff
@@ -13,6 +13,55 @@ GBD 2021 Modeling Strategy
 Cause Hierarchy
 +++++++++++++++
 
+- All causes (c_294) [level 0]
+
+  - Communicable, maternal, neonatal, and nutritional diseases (c_295)
+
+    - Maternal disorders and neonatal disorders (c_962)
+
+      - Maternal disorders (c_366)
+
+        - **Obstructed labor and uterine rupture (c_370)**
+
+            - Obstructed labor, acute event (s_188)
+
+            - Rectovaginal fistula (s_189)
+
+            - Vesicovaginal fistula (s_190)
+
+*Obstructed labor and uterine rupture (c_370)* is a most
+detailed cause, at level 4 of the GBD hierarchy. It has three sequelae,
+detailed in the following table:
+
+.. list-table:: Sequelae of obstructed labor and uterine rupture
+    :header-rows: 1
+    :widths: 2 1 5 5
+
+    * - Sequela
+      - GBD ID
+      - Health state and disability weight
+      - Notes
+    * - Obstructed labor, acute event 
+      - s_188
+      - abdominopelvic problem, severe 
+
+        DW: 0.324 (0.22–0.442) 
+      - 
+    * - Rectovaginal fistula 
+      - s_189
+      - rectovaginal fistula 
+
+        DW: 0.501 (0.339–0.657)
+      - The non-fatal burden of fistulas is included in the non-fatal 
+        burden of obstructed labour in reporting, but estimation is 
+        described in a separate appendix section on “Fistula – impairment.”
```

Comment:
It might look redundant, but for consistency's sake, I'd copy this note into the row for vesicovaginal fistula, too.

Also, I think it would be good to give a quick definition of each type of fistula in the notes column, as I'd never heard the word "fistula" until looking into this cause.